ai_chatbots/serializers.py

```diff
@@ -5,6 +5,16 @@
 
 from ai_chatbots.models import DjangoCheckpoint, LLMModel, UserChatSession
 
+MAX_MESSAGE_LENGTH = 6000
```

Comment:
would be good to move this to settings so its configurable via env var since I can see this being a thing we would want to change on the fly and/or keep consistent across multiple apps